bin/tinyreq

```diff
@@ -26,31 +27,36 @@ new clp({
   , examples: [
         "tinyreq -u http://ionicabizau.net"
     ]
-}, [urlOpt, methOpt, dataOpt, fieldsOpt]);
+}, [urlOpt, methOpt, dataOpt, fieldsOpt, quietOpt]);
 
 // Merge the options
 let options = ul.merge({
     method: methOpt.value || undefined
   , url: urlOpt.value || undefined
   , data: dataOpt.value || undefined
+  , quiet: quietOpt.value != 'false'
 }, {
     method: "GET"
 });
 
```

Comment:
You can do something like `logger.config.level = Infinity;` if the quiet option is provided.